server/src-lib/Hasura/RQL/DDL/Metadata.hs

```diff
@@ -214,6 +221,11 @@ applyQP2 (ReplaceMetadata tables templates) = do
       withPathK "delete_permissions" $ processPerms tabInfo $
         table ^. tmDeletePermissions
 
+    indexedForM_ tables $ \table -> do
```

Comment:
what is this code doing under `withPathK "queryTemplates"`

Reply:
expand the PR bruh